### DISCUSSION TITLE
Visus programs kill 1.6

### DISCUSSION THREAD

**Discussion by BilleBarrett** [Devolutions/UniGetUI#687] [General]
2 virus programs are flagging this
C:\Users\Gateway\AppData\Local\Programs\WingetUI\sudo\gsudo.exe

**Comment by marticliment** [comment]
have you checked virus total results?

**Reply by marticliment** [reply]
This file is taken from gerardog/gsudo, and is a required component for wingetui

**Comment by BilleBarrett** [comment]
This is occurring with site that do the upgrade to 1.60
Some are doing it some are not they are all running my end point BitDefender

 



My Staff 
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Reply by marticliment** [reply]
I will need you to submit your file to virustotal.

**Comment by Spacellary** [comment]
I use Windows Defender + Malwarebytes and never had it been detected. I also have my own version of gsudo from gerardog/gsudo and that also isn't being flagged.

**Comment by BilleBarrett** [comment]
I’m sorry I have 1500 clients on my monitoring services, I’m just uninstalling the software from the clients that are failing and maybe reinstall at a later date on a new update

Bill e. Barrett



 



My Staff 
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
